apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -603,6 +609,21 @@ NetSimLocalClientNode.prototype.onMessageTableChange_ = function() {
     // at us.  We must manually call a receive method instead.
     return;
   }
+  if (
+    // messageGranularity === BITS implies this is a simplex wire
+    NetSimGlobals.getLevelConfig().messageGranularity ===
+    MessageGranularity.BITS
+  ) {
+    this.getLatestMessageOnSimplexWire(
+      function(err, message) {
+        if (err) {
+          logger.error('Error pulling message off the wire: ' + err.message);
+          return;
+        }
+        this.wireStateChange.notifyObservers(message.payload);
+      }.bind(this)
+    );
```

Comment:
If we fall into this case, should we early-return? I believe the result of the logic in this method is specific to duplex-mode, and should be mutually exclusive with a simplex receive - it's a little bit of a coincidence that it's a no-op right now.